CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Fixed
+
+- `elasticsearch-discovery` service is not a headless service anymore
+- the `elasticsearch_memory_lock` variable was always considered true
```

Comment:
Maybe changing to:
```suggestion
- set a valid json value in `elasticsearch_memory_lock` variable
```

WDYT ?

Reply:
In fact, the configuration file in which this value is written is a YAML file (`elasticsearch.yml`).

According to the [YAML spec](https://yaml.org/type/bool.html), `False` should be a correct boolean value. But elasticsearch thinks it's `true`.

I used to_json to ensure that the value is lowercase, but I think I should have used `to_yaml` instead. The result is the same, but it will be less confusing.

So I'll follow your suggestion but write `yaml` instead of `json`.